net-analyzer/traceroute/traceroute-2.1.3.ebuild

```diff
@@ -16,6 +16,10 @@ IUSE="static"
 
 RDEPEND="!net-misc/iputils[traceroute6(-)]"
 
+src_configure() {
+	append-lfs-flags
```

Comment:
upstream has a bug tracker.  please file a bug with them and link that here.

Reply:
Done.

Btw, I've not fully understood why LFS is relevant with this package - diffing 2.1.0 and 2.1.3 I only see some additional usage of poll(). Is the LFS flags going to affect signature of poll()?

Reply:
CrOS has exceptions in place for specific versions to force people to clean things up the next time the package is upgraded